dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/AbstractTokenMapSupplier.java

```diff
@@ -227,13 +227,19 @@ private boolean isLocalDatacenterHost(Host host) {
                 String zone = (String) jItem.get("zone");
                 String datacenter = (String) jItem.get("dc");
                 String portStr = (String) jItem.get("port");
+                String securePortStr = (String) jItem.get("secure_port");
```

Comment:
Does DM currently expose the secure port?

Reply:
You reviewed it :)